src/cli/backup/exchange.go

```diff
@@ -108,25 +87,20 @@ func addExchangeCommands(cmd *cobra.Command) *cobra.Command {
 
 		// Flags addition ordering should follow the order we want them to appear in help and docs:
 		// More generic (ex: --user) and more frequently used flags take precedence.
-		fs.StringSliceVar(
-			&user,
-			utils.UserFN, nil,
-			"Backup Exchange data by a user's email; accepts '"+utils.Wildcard+"' to select all users")
+		utils.AddUserFlag(c)
+
 		fs.StringSliceVar(
 			&exchangeData,
 			utils.DataFN, nil,
 			"Select one or more types of data to backup: "+dataEmail+", "+dataContacts+", or "+dataEvents)
 		options.AddOperationFlags(c)
 
 	case listCommand:
-		c, fs = utils.AddCommand(cmd, exchangeListCmd())
-
-		fs.StringVar(&backupID,
-			"backup", "",
-			"ID of the backup to retrieve.")
+		c, _ = utils.AddCommand(cmd, exchangeListCmd())
```

Comment:
seems like you could remove the flag set return value from `AddCommand`?

Reply:
it's used in other places, primarily to maintain `fs.SortFlags = false`.  We should keep it in the return set for now.  Can always revisit later.